src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -125,13 +126,31 @@ The @ref PixelFormat::R8Unorm, @relativeref{PixelFormat,R8Srgb},
 @relativeref{PixelFormat,RGBA8Unorm} and @relativeref{PixelFormat,RGBA8Srgb}
 formats are supported.
 
+@subsection Trade-BasisImageConverter-behavior-types Image types
+
+1D images will be saved as 2D images with height 1.
+
+Cube map images can be written but there is currently no way to mark them
+properly in the metadata. Exported files will be 2D array images with faces
+exposed as layers.
+
+3D images will be saved as 2D array images with one layer per depth slice. This
+is necessary because mip levels are always 2-dimensional in Basis compressed
+images. Note that this matches @ref BasisImporter behavior which imports all 3D
+images as 2D array images anyway because of the same Basis limitation.
+
 @subsection Trade-BasisImageConverter-behavior-multilevel Multilevel images
 
 Images can be saved with multiple levels by using the list variants of
 @ref convertToFile() / @ref convertToData(). Largest level is expected to be
-first, with each following level having width and height divided by two,
+first, with each following level having width, height and depth divided by two,
 rounded down. Incomplete mip chains are supported.
```

Comment:
Wait, depth also? :eyes: 

```suggestion
first, with each following level having width and height divided by two,
rounded down. Because only 2D array images are supported, depth has to have the
same size in all levels. Incomplete mip chains are supported.
```